crates/chia-bls/src/gtelement.rs

```diff
@@ -13,7 +13,7 @@ use std::ops::{Mul, MulAssign};
     pyo3::pyclass,
     derive(chia_py_streamable_macro::PyStreamable)
 )]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
```

Comment:
I believe the `GTElement` is more than 500 bytes large. I think it's pretty good to require explicit cloning, to see where we pay that cost.